### ISSUE TITLE
The Cathedral: robe + red badge not working 

### ISSUE THREAD

**Issue by kafka2000** [rotators/Fo1in2#59]
I am at the Cathedral and Lasher gave me the red badge. I also have the required robe (from Boneyard). Normally the Nightkins should not attack me however they do even when i have the robe, the badge and i left my companions outside the cathedral (except for the dog, i cannot tell them to wait so i told they could leave).   

According to this [post](https://gamefaqs.gamespot.com/boards/197289-fallout/49560712) this is due to a bug in the game because of the dog. Is there any chance to fix the bug? Also i assume telling the characters to leave will have the same effect as them waiting (i.e. they will stay at the same place permanently and we can tell them to join us back even after many days).

**Comment by Lexx2k** [comment]
Uhm, you can tell Dogmeat to wait.

**Comment by Lexx2k** [comment]
Try leaving the companions in a different location. I'm curious if that will change something.

**Comment by kafka2000** [comment]
Sorry i was not clear: i had to tell my companions to leave me (not to wait) except for the dog whom I asked to wait. So while alone in the cathedral (without dog) this bug was still there.

I will try leaving them in a different city and report (i can also provide the saved game if useful).

**Comment by kafka2000** [comment]
> Uhm, you can tell Dogmeat to wait.
> 
> It's not something I will fix, because the Cathedral folks are suspicious of members with dog companions.
> tl;dr - not a bug.

tl;dr - the dog was waiting outside so isn't it a bug?

**Comment by kafka2000** [comment]
Ok so another bug: despite having 4 charisma + trait for 1 extra slot for companion, none of my 3 companions (Ian, Tycho, dog) want to join me back because the "group is too crowded". I attached the savegame if useful (illustrates both attack by Nightkins when alone with robe and bug of companions not rejoining).

[SLOT05_cathedral.zip](https://github.com/rotators/Fo1in2/files/4370976/SLOT05_cathedral.zip)

**Comment by kafka2000** [comment]
@Lexx2k I believe solving the "Nightkins' attack when all the characters are waiting outside (or another city)" bug would be important. This is one of the main quest and the bug unfortunately completely prevents the "pacific" approach to address it. This is a shame as the game clearly provide multiple clues to be able to accomplish this quest pacifically: robes found in Boneyard, badge given by Lasher, etc.

**Comment by Lexx2k** [comment]
I looked at your savegame ... you have the Follower scouts with you. Stealth is always impossible in that case.

Get rid of them with leaving the location first, or tell Nicole that you are better doing this alone.

This is also why you can't re-hire your party members. The Follower scouts count as party members. Same applies to the BoS paladins in Mariposa. It'll be fixed some day, probably.

**Comment by kafka2000** [comment]
Thank you very much for having a look at it @Lexx2k 

I had no idea about the scouts. So i assume there are right now in the cathedral as "hidden agents" and they will attack with me if i attack as well?

Well at least that explains everything, thanks a bunch again for having a look.
